app/components/post-search-results.js

```diff
@@ -0,0 +1,120 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { getQueryString } from '@wikia/ember-fandom/utils/url';
+
+import config from '../config/environment';
+
+// TODO: Remove this when all discussions' posts are in the index
+const QUIZZES_WHITELIST = [
+  'gameofthrones.fandom.com',
+  'attackontitan.fandom.com',
+  'marvelcinematicuniverse.fandom.com',
+  'marvel.fandom.com',
+  'southpark.fandom.com',
+  'starwars.fandom.com',
+  'strangerthings.fandom.com',
+  'xmenmovies.fandom.com',
+  'arrow.fandom.com',
+  'bojackhorseman.fandom.com',
+  'dc.fandom.com',
+  'dcextendeduniverse.fandom.com',
+  'disney.fandom.com',
+  'fastandfurious.fandom.com',
+  'godzilla.fandom.com',
+  'lionguard.fandom.com',
+  'miraculousladybug.fandom.com',
+  'riverdale.fandom.com',
+  'spongebob.fandom.com',
+  'tardis.fandom.com',
+  'thehungergames.fandom.com',
+  '13reasonswhy.fandom.com',
+  'battlefield.fandom.com',
+  'dragonage.fandom.com',
+  'acecombat.fandom.com',
+  'borderlands.fandom.com',
+  'pixelgun-wiki.fandom.com',
+];
+
+
+export default Component.extend({
+  fetchService: service('fetch'),
```

Comment:
This is so interesting. Just how this whole system works. Never touched ember.

Reply:
Yeah. It's a lot of :magic: